Eastern-Research-Group/mywaterway
# Feature/131 tribal page

This isn't 100% complete. We are still waiting on the attains release and for EPA to provide updated mapping files, which causes the Water Quality section to not have any data. I think it is good to go ahead and get this out on the dev site, so EPA can play with it and offer feedback. 

## Related Issues:
* https://jira.epa.gov/browse/HMW-131

## Main Changes:
* Added tribal data to the state page. The tribal page has a slightly different look than the state page.
  * Note: The Water Quality section currently doesn't have any data. This is because we are still waiting for the next attains release and we need EPA to provided updates to the `stateNationalUses.json` and `surveyMapping.json` files.
* Renamed all of the state files to `StateTribal...`

## Steps To Test:
1. Navigate to http://localhost:3000/state-and-tribal
2. Test state data
3. Test tribal data
4. Navigate directly to the following urls and verify they redirect back to http://localhost:3000/state-and-tribal
    * http://localhost:3000/state
    * http://localhost:3000/state/
    * http://localhost:3000/tribe
    * http://localhost:3000/tribe/
5. Test navigating directly to a state (i.e., http://localhost:3000/state/AR/water-quality-overview and http://localhost:3000/state/AR)
6. Test navigating directly to a tribe (i.e., http://localhost:3000/tribe/REDLAKE)
7. Test switching between tribes and states using the dropdown menu
8. Test full screen mode

## TODO:
- [ ] Retest after the attains release.
- [ ] Need to get updated versions of the `stateNationalUses.json` and `surveyMapping.json` files from EPA. This is why there isn't any data in the Water Quality section of the tribal page.
- [ ] Need to update the waterbody report to handle tribal data. I'll create a separate ticket for this. I need to look into this more, but we might not be able to work on this until after the attains release is complete.
